.github/workflows/ci.yml

```diff
@@ -92,6 +92,48 @@ jobs:
       uploadImageAsTarball: true
       platforms: linux/amd64
 
+  sdk-java:
+    name: Run SDK-Java integration tests
+    permissions:
+      contents: read
+      issues: read
+      checks: write
+      pull-requests: write
+      actions: read
+    secrets: inherit
+    needs: docker
+    uses: restatedev/sdk-java/.github/workflows/integration.yaml@main
+    with:
+      restateCommit: ${{ github.event.pull_request.head.sha || github.sha }}
+
+  sdk-python:
+    name: Run SDK-Python integration tests
+    permissions:
+      contents: read
+      issues: read
+      checks: write
+      pull-requests: write
+      actions: read
+    secrets: inherit
+    needs: docker
+    uses: restatedev/sdk-python/.github/workflows/integration.yaml@main
+    with:
+      restateCommit: ${{ github.event.pull_request.head.sha || github.sha }}
+
+  sdk-go:
+    name: Run SDK-Go integration tests
+    permissions:
+      contents: read
+      issues: read
+      checks: write
+      pull-requests: write
+      actions: read
+    secrets: inherit
+    needs: docker
+    uses: restatedev/sdk-go/.github/workflows/integration.yaml@main
+    with:
+      restateCommit: ${{ github.event.pull_request.head.sha || github.sha }}
```

Comment:
I guess in the future, we could only run the integration tests for a selected SDK and have the other SDKs tested against main on a scheduled basis if the number of Github action minutes becomes too large.